packages/venia-ui/lib/components/ProductOptions/tileList.js

```diff
@@ -6,25 +6,48 @@ import { useStyle } from '../../classify';
 import defaultClasses from './tileList.module.css';
 
 const TileList = props => {
-    const { getItemKey, selectedValue = {}, items, onSelectionChange } = props;
-
+    const {
+        getItemKey,
+        selectedValue = {},
+        items,
+        onSelectionChange,
+        isALLOutOfStock,
+        isVariantsOutOfStock
+    } = props;
+    let isOptionOutOfStock;
     const classes = useStyle(defaultClasses, props.classes);
 
     const tiles = useMemo(
         () =>
             items.map(item => {
                 const isSelected = item.label === selectedValue.label;
 
+                if (isVariantsOutOfStock && isVariantsOutOfStock.length > 0) {
+                    const flatOosArray = isVariantsOutOfStock.flat();
```

Comment:
Would update this to `flatOutOfStockArray`

Reply:
Thanks! It's updated.